ai-worker/Dockerfile

```
@@ -11,8 +11,11 @@ RUN pip install --no-cache-dir --prefix=/install .
 # ---------------------------------------------------------------------------
 FROM python:3.14-slim
 
-RUN groupadd --gid 1000 raven && \
-    useradd --uid 1000 --gid raven --create-home raven
+RUN apt-get update && apt-get install -y --no-install-recommends curl \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)